### PR TITLE
Update manage-security-groups.md

### PR DESCRIPTION
**Pull request by StephenRiceMSFT**
Updating to improve clarity around what types of groups are approved.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/manage-security-groups.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ As a SharePoint or global admin in Microsoft 365, you restrict external sharing
 
     ![Manage security groups](media/manage-security-groups.png)
 
-4. In the **Add a security group** box, enter a name for a security group. The security group box appears.
+4. In the **Add a security group** box, enter a name for a security group (Note that Microsoft 365 Groups are not supported). The security group box appears.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-security-groups.md:35]:
```suggestion
4. In the **Add a security group** box, enter a name for a security group. The security group box appears.

> [!NOTE]
> Microsoft 365 Groups are not supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestion.

File: SharePoint/SharePointOnline/manage-security-groups.md (hunk 1 above)
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-security-groups.md:35]:
```suggestion
4. In the **Add a security group** box, enter a name for a security group (note that Microsoft 365 Groups are not supported). The security group box appears.
```

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@StephenRiceMSFT, please commit this minor suggested change. Many thanks.

**Pull request merged**